### PR TITLE
I863 milestones chart points

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/goals/components/GoalsChart.js [hunk 1]
```diff
@@ -9,10 +9,15 @@ import {
 import GoalsChartActionsContainer from '../containers/GoalsChartActionsContainer';
 import MilestoneChartActionsContainer from '../containers/MilestoneChartActionsContainer';
 
-const getChartPoints = ({ _id: goalId, milestones }) => (
+const isOverdue = ({ completionTargetDate, isCompleted }) =>
+  completionTargetDate < new Date() && !isCompleted;
```
Comment (Dartv) [inline app/imports/ui/react/goals/components/GoalsChart.js]: Use `status` for that.

File: app/imports/ui/react/goals/components/GoalsChart.js [hunk 2]
```diff
@@ -9,10 +9,15 @@ import {
...
+
+const getChartPoints = ({ _id: goalId, milestones, color }) => (
   milestones.map(milestone => ({
     x: new Date(milestone.completionTargetDate),
     label: milestone.title,
+    fill: milestone.isCompleted ? color : '#fff',
```
Comment (Dartv) [inline app/imports/ui/react/goals/components/GoalsChart.js]: `status`

**Comment by Serhiy91** [comment]
@Dartv fixed

File: app/imports/ui/react/goals/components/GoalsChart.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const getChartPoints = ({ _id: goalId, milestones, color }) => (
     x: new Date(milestone.completionTargetDate),
     label: milestone.title,
     fill: milestone.status === GoalStatuses.COMPLETED ? color : '#fff',
-    stroke: milestone.status === GoalStatuses.OVERDUE ? color : '#dc3545',
+    stroke: milestone.status === GoalStatuses.OVERDUE ? '#dc3545' : color,
```
Comment (Dartv) [inline app/imports/ui/react/goals/components/GoalsChart.js]: use `MilestoneStatuses` instead

**Comment by Serhiy91** [comment]
fixed

**Pull request merged**